<launch>

<!-- Initial Paramteres -->
	<arg name="max_vel" default="2"/>
	<arg name="max_steer" default="1"/>


<!-- joy node -->
	<node respawn="true"
		pkg="joy"
		type="joy_node" 
		name="robot_joy" >
			
		<param name="dev" type="string" value="/dev/input/js0" />
		<param name="deadzone" value="0.001" />
		<param name="autorepeat_rate" type="double" value="0" />
		<remap from="joy" to="/joy"/> 
  	</node>


<!-- Teleoperete node -->
	<node name="teleop_joy" 
    	  pkg="turtle_infnet" 
		  type="teleopJoy.py"
		  args="$(arg max_vel) $(arg max_steer)"
		  >
		    
		<remap from="cmd_vel" to="/turtle1/cmd_vel"/>
		<remap from="joy" to="/joy"/>

	</node>

</launch>
